atlasdb-ete-tests/build.gradle

```diff
@@ -77,7 +77,7 @@ distribution {
     serviceName 'atlasdb-ete'
     mainClass 'com.palantir.atlasdb.AtlasDbEteServer'
     args 'server', 'var/conf/atlasdb-ete.yml'
-    defaultJvmOpts '-Xmx384M'
+    defaultJvmOpts '-Xmx768M'
```

Comment:
huhhh?

Reply:
Ete tests were ooming the container when all those lock descriptors were created and serialized